tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md

```diff
@@ -38,11 +37,17 @@ pip --version
 
 [ACCORDION-BEGIN [Step 2: ](Install SAP AI API Client SDK)]
 
-Python packages have format `.whl` or `.tar.gz`, both can be installed in the same way. `SAP-AI-API-Client-SDK`  is python package which will enable us to talk to SAP AI API.
-Follow the steps to install the same
+`SAP-AI-API-Client-SDK`  is python package which will enable us to talk to SAP AI API.
```

Comment:
Please don't use the "royal we" unless you are referring to SAP. Say "you" instead when you are addressing the user.

`SAP-AI-API-Client-SDK`  is python package - There is a missing article and there should be a comma before "which".